src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -173,55 +175,59 @@ export class SchoolSelectField extends React.Component {
               </button>
             </div>
           </div>
-          {showInstitutions && <div>
-            {institutions.map(({ city, facilityCode, name, state }, index) => (
-              <div key={index}>
-                <div className="radio-button">
-                  <input
-                    autoComplete="false"
-                    checked={facilityCodeSelected === facilityCode}
-                    id={`radio-buttons-${index}`}
-                    name={name}
-                    type="radio"
-                    onKeyDown={this.onKeyDown}
-                    onChange={() => this.handleOptionClick({ city, facilityCode, name, state })}
-                    value={facilityCode}/>
-                  <label
-                    id={`institution-${index}-label`}
-                    htmlFor={`radio-buttons-${index}`}>
-                    <span className="institution-name">{name}</span>
-                    <span className="institution-city-state">{`${city}, ${state}`}</span>
-                  </label>
-                </div>
-              </div>))
+          <div
+            aria-live="polite"
+            aria-relevant="additions text">
```

Comment:
aria tags for a live updating section. This has to be rendered when the component is mounted